### DISCUSSION TITLE
Pre-release 2.0 beta available for testing!

### DISCUSSION THREAD

**Discussion by jorisvandenbossche** [shapely/shapely#1578] [Announcements]
Please see the new [migration guide](https://shapely.readthedocs.io/en/latest/migration.html) and the [detailed changelog](https://shapely.readthedocs.io/en/latest/release/2.x.html#version-2-0-0) for 2.0 and try this out if you can. The project will be grateful for any feedback you care to provide in this discussion thread.

To install shapely 2.0 beta 1, you can do (recommended to this in a new environment):

```bash
# using pip
pip install --pre shapely==2.0b1
# using conda /mamba
conda install -c conda-forge/label/shapely_dev shapely
```

**Comment by vincentsarago** [comment]
@jorisvandenbossche I've switched from pygeos to shapely 2.0 in https://github.com/developmentseed/cogeo-mosaic and it works perfectly (I assume you're looking for more feedback from people that were already using Shapely 😅)

Do you have an sense when 2.0 will be officially released?

**Reply by jorisvandenbossche** [reply]
Great to hear, thanks for testing 2.0!

> Do you have an sense when 2.0 will be officially released?

https://github.com/shapely/shapely/issues/962 has the latest details. I hope to release a RC1 in one of the next few days, and then a final release 1-2 weeks later.
